tests/chainer_tests/functions_tests/normalization_tests/test_batch_normalization.py

```diff
@@ -164,6 +164,157 @@ def test_double_backward(self, backend_config):
             backend_config)
 
 
+@testing.parameterize(*testing.product_dict(
```

Comment:
Could you merge these tests into `TestBatchNormalization`?
How about `if self.axis is None`, after concatenating the parameter sets and adding `'axis': None` to the older sets.

Reply:
Do you mean I should support `axis` option for ideep? I have no idea how to do that and I'm not appropriate person to do that.

Reply:
No.  My concern was just that most of the lines of `TestBatchNormalizationAxis` are copied from `TestBatchNormalization`.  I'm willing to work on this issue as another PR.